PZ-3913 - Adding info about remainder payment details in guides (#671)
mollie/api-documentation: source/guides/gift-cards.rst

You cannot perform any gift card refunds. However, if another payment method was used during the checkout, you can
refund the part paid with the other payment method (and optionally an additional part).

Getting the details of the remainder payment
--------------------------------------------
In some cases it is desirable to receive the details of a payment. This is mostly the case from the

> remainder payment when the consumer partially paid with a gift card. Receiving these details is possible

Reviewer: ...consumer pays with both a gift card and another payment method.

in both the :doc:`Payments API </reference/v2/payments-api/get-payment>` and
:doc:`Orders API </reference/v2/orders-api/get-order>`.

To receive the remainder details in the Payments API, use the ``details.remainderDetails`` include.
For example: ``GET https://api.mollie.com/v2/payments/tr_xxx?include=details.remainderDetails``. For
the Orders API you should the ``payments.details.remainderDetails`` embed;
``GET https://api.mollie.com/v2/orders/ord_xxx?embed=payments.details.remainderDetails``.

> Be aware that the ``remainderDetails`` is only available when the payment ws partially paid with

Reviewer 2: Typo: "the payment **ws** partially paid"

> Be aware that the ``remainderDetails`` is only available when the payment was not fully paid with a
> gift card.

Reviewer 3: ```suggestion
Be aware that the ``remainderDetails`` is only available when the payment is partially paid with a gift card.
```

> a gift card and then the remainder was paid with a different payment method.

Reviewer 2: What happens if you use `reminderDetails` in a non partially paid payment? Do get an empty object, or no object at all?

Author: No object at all
